nav_msgs/OccupancyGrid[] desirability_maps